src/style.css

```diff
@@ -1,3 +1,115 @@
+/* Configuração geral da fonte */
 body {
-  margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font-family in case the main one doesn't load. For example, 'Roboto', sans-serif could have a fallback to a generic font family like Arial.